### PR TITLE
feat: add memcache_version parameter

### PR DESCRIPTION
**Pull request by bruno561**
I propose that we add the memcached version parameter.
Doc: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/memcache_instance#memcache_version

Related issue: #184

### CODE REVIEW

File: modules/memcache/variables.tf [hunk 1]
```diff
@@ -35,6 +35,12 @@ variable "name" {
   type        = string
 }
 
+variable "memcache_version" {
+  description = "The major version of Memcached software."
+  type        =
```
Comment (imrannayer) [inline modules/memcache/variables.tf:40]: plz define type.

Comment (bruno561) [reply modules/memcache/variables.tf]: Done! @imrannayer

**Comment by bruno561** [comment]
@imrannayer asap, can you review again please?

**Comment by imrannayer** [comment]
@bruno561 can u plz fix lint issue?

**Comment by bruno561** [comment]
> @bruno561 can u plz fix lint issue?

Can you give more context of the error so I can adjust? I didn't find much information in the run output.

**Comment by imrannayer** [comment]
@bruno561 plz follow [this doc](https://github.com/terraform-google-modules/terraform-google-memorystore/blob/master/CONTRIBUTING.md)

**Comment by bruno561** [comment]
@imrannayer I added this new [commit](https://github.com/terraform-google-modules/terraform-google-memorystore/pull/185/commits/141c4f89d2f406cfefba9fed13181f23ec602a47) and ran make `docker_test_lint` local and I didn't get any more errors.

**Review by imrannayer — APPROVED** [review]

**Pull request merged**